src/_test_/Currency.test.js

```diff
@@ -0,0 +1,32 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import myCurrencies from './Currencies';
+import Currency from '../component/Currency';
+
+const reducer = (
+  state = {
+    currencies: { currencies: myCurrencies },
+  },
+) => state;
+
+const store = configureStore({ reducer });
+describe('Render Test', () => {
+  it('Component renders correctly', () => {
+    const data = {
+      name: 'argentina',
+      symbol: 'aug',
+    };
+    const mytree = render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <Currency eachCurrency={data} />
+        </Provider>
+      </MemoryRouter>,
+    );
+    expect(mytree).toMatchSnapshot();
+  });
```

Comment:
- [ ] Good job creating some unit tests , but as part of the requirements kindly create some Unit tests as well 🙏